README.md

```diff
@@ -1,45 +1,79 @@
 
 ```rust
 #[derive(GraphQLQuery)]
 #[gql(
-    query = "/graphql/queries/my_query.graphql",
-    schema = "/graphql/schema.graphql"
+    // The paths are relative to the directory where your `Cargo.toml` is located.
+    query_path = "src/graphql/queries/my_query.graphql",
+    schema_path = "src/graphql/schema.json"
 )]
 struct MyQuery;
+```
 
+```rust
 fn perform_my_query(variables: &my_query::Variables) -> Result<(), failure::Error> {
     let body = MyQuery::expand(variables);
     let client = reqwest::Client::new();
-    let res: HttpResponse<graphql_client::Response<my_query::ResponseData>> = client.post("/graphql", body)?;
-    println!("{:#?}", res.body);
+    let mut res: HttpResponse<graphql_client::Response<my_query::ResponseData>> = client.post("/graphql").json(&body).send()?;
+    println!("{:#?}", res.text());
+    Ok(())
 }
 ```
 
+The GraphQL schema language (`.graphql`) and `schema.json` are both supported as sources for the schema.
+
+`serde_derive` needs to be visible in the context of the `GraphQLQuery` derive (add it as an `extern crate`).
+
 ## Features
 
-* Strongly typed query variables
-* Strongly typed response
+- Strongly typed query variables
+- Strongly typed responses
+- Works in the browser (WebAssembly)
+
+Integration with different HTTP libraries is planned, although building one yourself is trivial (just send the constructed request payload as JSON with a POST request to a GraphQL endpoint, modulo authentication).
 
-### Planned features
+There is an embryonic CLI for downloading schemas - the plan is to make it something similar to `apollo-codegen`.
 
-* Strongly typed subscriptions
-* Query string minification (e.g. for embedding in a browser wasm app, and for minimizing payload size)
-* A command line interface in addition to the custom derive for generating code and downloading schemas
 
-## What is generated?
+## What is generated?
 
-* A module named after the struct under derive, which contains:
-  * A `ResponseData` struct implementing `serde::Deserialize`
-  * A `Variables` struct meant to contain the variables expected by the query
-* An impl for the `GraphQLQuery` trait for the struct under derive
+- A module named after the struct under derive, which contains:
+  - A `ResponseData` struct implementing `serde::Deserialize`
+  - A `Variables` struct meant to contain the variables expected by the query
+- An impl for the `GraphQLQuery` trait for the struct under derive
 
-See the example generated module for a full example.
+See the [example generated module](https://www.tomhoule.com/docs/example_module/) for more details.
 
 ## Examples
 
 See the examples directory in this project.
+
+## Code of conduct
+
+Anyone who interacts with this project in any space, including but not limited to
+this GitHub repository, must follow our [code of conduct](https://github.com/tomhoule/graphql-client/blob/master/CODE_OF CONDUCT.md).
```

Comment:
Link is broken because of the missing underscore

Reply:
thanks!